src/sig_stfl/lms/external/hash.c

```diff
@@ -32,7 +33,8 @@ void hss_hash_ctx(void *result, int hash_type, union hash_context *ctx,
           const void *message, size_t message_len) {
 #if ALLOW_VERBOSE
     if (hss_verbose) {
-        int i; for (i=0; i< message_len; i++) printf( " %02x%s", ((unsigned char*)message)[i], (i%16 == 15) ? "\n" : "" );
+        int i; for (i=0; i< message_len; i++) printf( " %02x%s", ((unsigned char*)message)[i], (i%16 == 15) ? "
```

Comment:
Did something go wrong with the `\n` symbols here?

Reply:
Yep, will fix.